### PR TITLE
Support `HIGH`, `LOW`, `BITWIDTH`, and `TZCOUNT` RPN operations

### PR DESCRIPTION
**Pull request by Rangi42**
These will be added in object file format v6 r11, with RGBDS v0.9.0. (So we'd also release rgbds-obj v0.3.0 and rgbobj v0.4.0.)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -22,7 +23,7 @@
 //! [v0.4.0](https://rgbds.gbdev.io/docs/v0.4.0/rgbds.5)   | v9 r3
 //! [v0.3.10](https://rgbds.gbdev.io/docs/v0.3.10/rgbds.5) | v6
 //!
-//! Earlier releases use v6, up to and including v0.3.4.
+//! RGBDS releases v0.3.4 and earlier use object format v6.
```
Comment (ISSOtm) [inline src/lib.rs:26]: I think this was rather meaning to say that v6 lasted *from* 0.3.4, until 0.3.10.

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Two tiny doc nitpicks on an otherwise straightforward PR. Thanks!

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rgbds-obj"
 version = "0.2.1" # Remember to sync html_root_url in `lib.rs`
-authors = ["ISSOtm <me@eldred.fr>"]
+authors = ["ISSOtm <me@eldred.fr>", "Rangi42 <https://github.com/Rangi42>"]
```
[the remark this answers is not in the document]
Comment (Rangi42) [reply Cargo.toml:4]: Okay, at least an email is [optional](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field).

**Review by ISSOtm — APPROVED** [review]
Thank you§

**Pull request merged**